MapboxNavigation/RouteManeuverViewController.swift

```diff
@@ -170,12 +170,12 @@ class RouteManeuverViewController: UIViewController {
     }
     
     func updateStreetNameForStep() {
-        if let step = step, step.isNumberedMotorway, let codes = step.codes {
+        if let destinations = step?.destinations {
+            destinationLabel.unabridgedText = destinations.joined(separator: NSLocalizedString("DESTINATION_DELIMITER", bundle: .mapboxNavigation, value: " / ", comment: "Delimiter between destinations in a road concurrency"))
```

Comment:
This change could cause the label’s font size to shrink beyond what it normally used to. At some point, would it be more readable to allow a second line of destination text?